compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -2090,14 +2090,14 @@ fn generic_simd_intrinsic<'ll, 'tcx>(
         };
     }
 
-    arith_red!(simd_reduce_add_ordered: vector_reduce_add, vector_reduce_fadd, true, add, 0.0);
+    arith_red!(simd_reduce_add_ordered: vector_reduce_add, vector_reduce_fadd, true, add, -0.0);
```

Comment:
What does the value even do here? simd_reduce_add_ordered takes a start value, it shouldn't need another start value set by codegen...

Reply:
You're right, it does nothing, the macro just isn't very resilient.

Reply:
I poked at it slightly and I have determined that it isn't really worth refactoring the macro to allow eliding this argument without doing a LOT more work on it, heh.